ceph/ceph_admin/bootstrap.py

```diff
@@ -231,7 +231,9 @@ def bootstrap(self: CephAdmProtocol, config: Dict):
             #       used in the CI test modules.
             #       The `set_cdn_tool_repo` should be taken out, once
             #       we have upstream build with all necessary pkg sources.
-            self.set_cdn_tool_repo(_rhcs_version)
+            #       [Oct-2024]: Fixed by adding EPEL public repo, cdn repo should no
+            #                   longer be needed
+            # self.set_cdn_tool_repo(_rhcs_version)
```

Comment:
Please remove these comments. A proper commit message should be sufficient. The request is primarly because we are commenting the call to the method.

Reply:
Removed all the comments, new as well old to-do